sound_play/scripts/soundplay_node.py

```diff
@@ -254,11 +254,15 @@ def select_sound(self, data):
                 voice = data.arg2
                 try:
                     try:
-                        txtfile.write(data.arg.decode('UTF-8').encode('ISO-8859-15'))
+                        if hasattr(data.arg, 'decode'):
+                            txtfile.write(data.arg.decode('UTF-8').encode('ISO-8859-15'))
+                        else:
+                            txtfile.write(data.arg.encode('ISO-8859-15'))
                     except UnicodeEncodeError:
-                        txtfile.write(data.arg)
-                    except AttributeError:
-                        txtfile.write(data.arg.encode('ISO-8859-15'))
+                        if hasattr(data.arg, 'decode'):
+                            txtfile.write(data.arg.decode('UTF-8').encode('UTF-8'))
```

Comment:
Does this line have any effect ?

Reply:
yes, you are right.
thank you, I updated.